### DISCUSSION TITLE
My vscode extension showing as 'not available in Visual Studio Code for the Web' despite adding browser entry point

### DISCUSSION THREAD

**Discussion by jonathanyeung** [community/community#28989] [Codespaces]
I just upgraded my extension to try to get it to work on vscode on the web. I added a new entry point for 'browser' in my package.json.

When viewing in the extensions tab in the browser, my extension now no longer shows the yellow exclamation mark of browser incompatibility and I get an option to 'Install', so it _seems_ like something worked.  But when I actually try to install it, I get an error: 

<img width="1123" alt="image" src="https://user-images.githubusercontent.com/3203268/183872789-fabdcf87-d9a1-4ac9-82c5-5d39fc8ec2dd.png">

From my local testing + opening up the .vsix file, the 'package.json' manifest and bundle seem to configured correctly. It appears that the extension is being gated on the Marketplace side - any ideas or pointers? 

The extension in question is latest version (v107.1) of https://marketplace.visualstudio.com/items?itemName=dendron.nightly 

Thanks!

**Comment by joshaber** [comment]
Hi @jonathanyeung, I'd recommend reaching out to VS Code in https://github.com/microsoft/vscode. They'll be able to better help with questions around extensions & packaging.

**Reply by jonathanyeung** [reply]
@joshaber - thanks for the tip.  I wasn't sure where to put it, I've since created a new issue here: https://github.com/microsoft/vscode/issues/157805
